.github/workflows/scripts/config-diff-post-comment.js

```diff
@@ -0,0 +1,61 @@
+module.exports = async ({ github, context, core, configDiff }) => {
+    try {
+        // Do not create comment if there are no configuration changes
+        if (!configDiff) {
+          console.log("No changes detected. Skipping comment creation.");
+          return;
+        }
+    
+        const commentBody = `
+### Config Diff Tool Output
+
+\`\`\`diff
+
+${configDiff}
+
+\`\`\`
+  
+   
+The above configuration changes are found in the PR. Please update the relevant release documentation if necessary.
+    `;
+    
+        const { owner, repo } = context.repo;
+        const issueNumber = context.payload.pull_request.number;
+    
+        const comments = await github.paginate(
+          github.rest.issues.listComments, {
+            owner,
+            repo,
+            issue_number: issueNumber,
+            per_page: 100,
+          }
+        );
+    
+        const existingComment = comments.find(comment => comment.body.includes("### Config Diff Tool Output"));
+    
+        if (existingComment) {
+          console.log("A config diff comment already exists, deleting it...");
+          // Update the existing comment
+          await github.rest.issues.deleteComment({
+            comment_id: existingComment.id,
+            owner,
+            repo,
+          });
+        }
```

Comment:
I'm not sure about deleting comments. I see there might other ways to provide that feedback. As a reference, [the ReviewDog Action shows the different formats](https://github.com/reviewdog/reviewdog#reporter-github-pr-checks--reportergithub-pr-check) that can be used to report issues.

Reply:
BTW, I just found the `core` [API for setting `annotations`](https://github.com/actions/toolkit/tree/main/packages/core#annotations).

Reply:
Thank you, have now fixed it and we display annotations. I created a branch from this PR and ran the worklow using `pull_request` and the annotations pop up. Look here: [1](https://github.com/ceph/ceph/pull/63098/files#diff-24521e8b04f166f7cfcd9bccf49c69817e42a1681fa98f1a8af11893413adedc) and [2](https://github.com/ceph/ceph/pull/63098/files#diff-9dea9904bbc6ee3be2b9ffb98b234de2f71066b559b9dc0cc0fe305cdbc6580b)

The action summary is here: https://github.com/ceph/ceph/actions/runs/15422801975

They look like this: 
![image](https://github.com/user-attachments/assets/97479568-6d0c-493a-8023-6987a1ff2480)